src/hooks/useFetchData.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from "react";
+import fetchData from "../apis/api";
+
+export default function useFetchData(apiEndpoint) {
```

Comment:
```suggestion
export default function useFetchData(endpoint) {
```
어차피 data fetch할떄는 api를 요청할거니까 endPoint로만 적어도 될 것 같아요!